Routing/RestRouteCollection.php

```diff
@@ -76,10 +76,17 @@ public function setDefaultFormat($format)
      */
     public function all()
     {
+        $regex = <<<REGEX
```

Comment:
ok, then I'll change it. I observed that syntax in some parts of symfony, so I assume, it would be ok to use it (https://github.com/symfony/debug/blob/master/ExceptionHandler.php#L272)